src/utilities/GenerateTemplateFiles.batch.test.ts

```diff
@@ -0,0 +1,55 @@
+import GenerateTemplateFiles from '../GenerateTemplateFiles';
+import { IConfigItem } from '../index';
+import CaseConverterEnum from '../constants/CaseConverterEnum';
+
+describe.skip('GenerateTemplateFiles - Batch', () => {
```

Comment:
Should `.skip` be removed?

```suggestion
describe('GenerateTemplateFiles - Batch', () => {
```

Reply:
Ah whoops, copypasta derm. Should I unlock the commandLine tests as well?